torchmetrics/functional/text/eed.py

```diff
@@ -0,0 +1,446 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# referenced from
+# Library Name: torchtext
+# Authors: torchtext authors
+# Date: 2021-12-07
+# Link:
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# The RWTH Extended Edit Distance (EED) License
```

Comment:
just wondering how much is the code similar to the original implementation?
https://github.com/rwth-i6/ExtendedEditDistance/blob/master/EED.py
if we would just follow the paper we do not need this long copy licence and just refer the paper (I guess)

Reply:
@justusschock ^^